library/core/src/macros/mod.rs

```diff
@@ -198,14 +198,11 @@ pub macro assert_matches {
 
 /// A macro for defining `#[cfg]` match-like statements.
```

Comment:
Given that we renamed it since it ended up not seeming enough like a `match` to us, probably we should reword this as well.  Maybe:

```suggestion
/// Selects code at compile-time based on `cfg` predicates.
```